css/types/shape.json

```diff
@@ -0,0 +1,59 @@
+{
+  "css": {
+    "types": {
+      "shape": {
+        "__compat": {
```

Comment:
We should have a subfeature rect() in addition to that. <shape> is intended in the future to have more value than rect().
The compat info for <shape> and rect() is the same, with the IE note on rect(), though.

Reply:
The MDN page says that `<shape>` is being deprecated in favor of `<basic-shape>`. Is that incorrect? Should I update the status of `<shape>` too?

Reply:
Ok, clip is deprecated so <shape> will not be extended. But we still need the subfeature. We can mark shape as deprecated.

Reply:
OK, I've update shape with a `rect()` feature. Let me know if you want to see other changes. Thanks!